index.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+const BroccoliDebug = require('broccoli-debug');
+const BroccoliMergeTrees = require('broccoli-merge-trees');
+
+const Prerender = require('./lib/prerender');
```

Comment:
It was intentional that we `require`'d these things lazily on demand.

By requiring them up front, people running things like `ember help` or `ember generate` are forced to parse and evaluate all those dependencies even though they won't use them.